docker-compose.dev.yaml

```diff
@@ -1,38 +1,73 @@
 version: '3'
 
 services:
-  content-publishing-service:
-    build:
-      context: .
-      dockerfile: dev.Dockerfile
-    ports:
-      - 3000:3000
-    env_file:
-      - .env.docker.dev
-    volumes:
-    - ./:/app
-    networks:
-      - content-publishing-service-network-dev
-
   redis:
     image: redis:latest
     ports:
       - 6379:6379
-    networks:
-      - content-publishing-service-network-dev
     volumes:
       - redis_data:/data
+    networks:
+      - content-publishing-service
 
   frequency:
     image: frequencychain/instant-seal-node:latest
     ports:
       - 9944:9944
     networks:
-      - content-publishing-service-network-dev
-    container_name: frequency-node
+      - content-publishing-service
+
+  kubo_ipfs:
```

Comment:
does kubo need volume?, it looks like since it is storing the files it might need some volume

Reply:
added volume for all